attachment_unindex_content/i18n/attachment_unindex_content.pot

```diff
@@ -0,0 +1,20 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* attachment_unindex_content
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 12.0\n"
```

Comment:
11.0

Reply:
Better, export it directly from odoo 11.0 in order to reduce the diff for the next change from translate bot

Reply:
Done !